solar_installation/data/sale_order_template_line.xml

```diff
@@ -0,0 +1,159 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="sale_order_template_line_1" model="sale.order.template.line">
+        <field name="name">Solar Installation Charge</field>
+        <field name="product_id" ref="product_product_5"/>
+        <field name="sale_order_template_id" ref="sale_order_template_2"/>
+    </record>
+    <record id="sale_order_template_line_2" model="sale.order.template.line">
+        <field name="name">Solar Panel set</field>
```

Comment:
If the name is the same as the product, there is no need to define it again here
```suggestion
```

Reply:
Can't it be applied to some other lines as well?

Reply:
No we need it

Reply:
Yes its done